code/modules/xenoarchaeology/xenoartifact_labeler.dm

```diff
@@ -0,0 +1,217 @@
+/obj/item/xenoartifact_labeler
+    name = "Xenoartifact Labeler"
+    icon = 'icons/obj/xenoarchaeology/xenoartifact_tech.dmi'
+    icon_state = "xenoartifact_labeler"
+    desc = "A tool scientists use to label their alien bombs."
+    throw_speed = 3
+    throw_range = 5
+    w_class = WEIGHT_CLASS_TINY
+
+    var/list/activator = list(null) //Checked trait
+    var/list/activator_traits = list() //Display names
+
+    var/list/minor_trait = list(null, null, null)
+    var/list/minor_traits = list()
+
+    var/list/major_trait = list(null)
+    var/list/major_traits = list()
+
+    var/list/malfunction = list(null)
+    var/list/malfunction_list = list()  
```

Comment:
Why add null to some of these lists?

Reply:
I believe it was to do with sizing. I'm under the impression changing it breaks it.